go.mod

```diff
@@ -98,7 +98,7 @@ require (
 )
 
 require (
-	github.com/cockroachdb/pebble v0.0.0-20230829171545-74d4bb649d92
+	github.com/cockroachdb/pebble v0.0.0-20230906160148-46873a6a7a06
```

Comment:
we have to update this one to not cause import ambiguity (EVM imports some dependencies that conflict with this version)